packages/plugins/plugins/plugin-aws/src/blocks/S3UploadPhoto/S3UploadPhoto.js

```diff
@@ -122,23 +44,30 @@ const S3UploadPhoto = ({ blockId, components: { Icon }, events, methods, propert
       ],
     });
   }, []);
-
+  useEffect(() => {
+    if (JSON.stringify(value) !== JSON.stringify(state)) {
+      setValue(value);
+    }
+  }, [value]);
   return (
     <Upload
       listType="picture-card"
       className="avatar-uploader"
       accept="image/*"
-      customRequest={customRequest}
-      maxCount={1}
+      customRequest={s3UploadRequest}
+      maxCount={properties.maxCount}
```

Comment:
Please add changeset for this new property. I also don't see it in the schema.json